digitalaidseattle/seattle-humane-app
# Features/1.2.2.2.21/use team member hook

## What type of PR is this? (check all applicable)


## Description

Adds the hook for UI elements that lookup up team members

## Related Tickets & Documents

Related to the input form e2e scenario

## QA Instructions, Screenshots, Recordings

1. `yarn install && yarn dev` 
1. Open localhost:3000
1. Click new request 
1. Verify that Assigned To dropdown allows selecting email address(es) 
 

## Added/updated tests?
_We are aiming for 100% statement coverage._

      have not been included_
